JudoPayDotNet/Http/Client.cs

```diff
@@ -6,46 +6,46 @@ namespace JudoPayDotNet.Http
 {
     internal class Client : IClient
     {
-        private readonly Connection _connection;
+        public Connection Connection { get; }
```

Comment:
does this really need to be public? if you just need it in tests it can be internal

Reply:
This needs to be described on the IClient interface as an IClient is used in the tests.

Reply:
@James-Lappin we already have InternalsVisibleTo set for the tests project, so internal would work